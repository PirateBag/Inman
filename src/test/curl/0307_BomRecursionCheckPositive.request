{ "parentId" : "2", "childId" : "1"  }

