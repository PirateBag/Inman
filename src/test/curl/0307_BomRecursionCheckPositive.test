Empty Test File
=== JSON ===
{ "parentId" : "2", "childId" : "1"  }

