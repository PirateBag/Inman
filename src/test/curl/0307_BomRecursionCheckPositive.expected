{
  "responseType" : "QUERY",
  "errors" : [ {
    "key" : 1,
    "message" : "Parent and proposed child are same item.  1"
  } ],
  "data" : [ ]
}